src/java.base/share/classes/sun/nio/ch/FileChannelImpl.java

```diff
@@ -529,6 +530,46 @@ public long size() throws IOException {
         }
     }
 
+    public int available() throws IOException {
```

Comment:
This can be package-private. It would be useful to add a method description as FC doesn't define this method, same thing for isOther.

Reply:
Fixed in a163d8d.